tests/catch.hpp

```diff
@@ -3556,7 +3556,7 @@ namespace Matchers {
     StdString::EqualsMatcher Equals( std::string const& str, CaseSensitive::Choice caseSensitivity = CaseSensitive::Yes );
```

Comment:
Don't change this file. This is a file that comes from another project. We should not change it.